components/NavLink.js

```diff
@@ -0,0 +1,14 @@
+import Link from "next/link";
+import React from "react";
+
+function NavLink({ href, children, ...props }) {
```

Comment:
Poți extrage și alte props de care e posibil să ai nevoie la un moment dat. ([docs](https://nextjs.org/docs/api-reference/next/link)).

```suggestion
function NavLink({ href, children,  prefertch = true, replace = false, scroll = true, shallow = false, locale = true, ...props }) {
```